front/pages/w/[wId]/builder/assistants/new.tsx

```diff
@@ -70,9 +126,52 @@ export default function CreateAssistant({
   plan,
   gaTrackingId,
   dataSources,
+  dataSourceConfigurations,
   dustApps,
+  dustAppConfiguration,
+  databaseQueryConfiguration,
+  agentConfiguration,
   flow,
 }: InferGetServerSidePropsType<typeof getServerSideProps>) {
+  let actionMode: AssistantBuilderInitialState["actionMode"] = "GENERIC";
+
+  let timeFrame: AssistantBuilderInitialState["timeFrame"] = null;
+
+  if (agentConfiguration) {
```

Comment:
Same logic as in the edit file, but guarded by the existence of an `agentConfiguration`